src/dispatchers/dispatcher.ts

```diff
@@ -30,21 +31,14 @@ module Plottable {
       }
     }
 
-    /**
-     * Creates a wrapped version of the callback that can be registered to a Broadcaster
-     */
-    protected _getWrappedCallback(callback: Function): Core.BroadcasterCallback<Dispatcher> {
-      return () => callback();
+    protected _setCallback(callbackSet: Utils.CallbackSet<Function>, callback: Function) {
+      this._connect();
+      callbackSet.add(callback);
     }
 
-    protected _setCallback(b: Core.Broadcaster<Dispatcher>, key: any, callback: Function) {
-      if (callback === null) { // remove listener if callback is null
-        b.deregisterListener(key);
-        this._disconnect();
-      } else {
-        this._connect();
-        b.registerListener(key, this._getWrappedCallback(callback));
-      }
+    protected _unsetCallback(callbackSet: Utils.CallbackSet<Function>, callback: Function) {
+      callbackSet.delete(callback);
+      this._disconnect();
```

Comment:
Based on our switching of API names, do you think we should change these names?  Not a huge deal.